src/app/cli/src/output/verify_progress.rs

```diff
@@ -54,6 +54,10 @@ impl VerificationMultiListener for VerificationMultiProgress {
             self.multi_progress.clone(),
         )))
     }
+
+    fn begin_multi_verify(&self) -> Option<Arc<dyn VerificationMultiListener>> {
+        Some(Arc::new(VerificationMultiProgress::new()))
```

Comment:
Not sure why you need an existing `VerificationMultiProgress` to construct a new `VerificationMultiProgress` instance here.